components/mailbox/src/Mailbox.svelte

```diff
@@ -640,20 +644,19 @@
     }
     draftMessageUpdate(message);
 
-    const value = {
-      to: message.to,
-      cc: message.cc,
-      bcc: message.bcc,
-      from: message.from,
-      subject: message.subject,
-      body: message.body,
-    };
+    dispatchEvent("draftThreadClicked", {
+      event,
+      message,
+      thread,
+    });
+
```

Comment:
Reviewers, for the deprecated `draftThreadEvent` should I remove it completely here?

Reply:
Is it possible to throw a warning _if_ used? But since we're not making a BC change, it will need to stay.